File: MiniDOMWebCtrls/DMD/CQSPD/CGruppoProdottiModuleHandler.vb
Imports minidom
Imports minidom.Forms
Imports minidom.Databases
Imports minidom.WebSite

Imports minidom.Finanziaria
Imports minidom.Sistema
Imports minidom.Anagrafica

Namespace Forms
  
    Public Class CGruppoProdottiModuleHandler
        Inherits CBaseModuleHandler

        Public Sub New()
            
        End Sub

        Public Overrides Function CreateCursor() As DBObjectCursorBase
            Dim cursor As New CGruppoProdottiCursor
            Return cursor
        End Function


    End Class




End Namespace

File: MiniDOMWebCtrls/DMD/CQSPD/CGruppoProdottiModuleHandler.cs

namespace minidom.Forms
{
    public class CGruppoProdottiModuleHandler : CBaseModuleHandler
    {
        public CGruppoProdottiModuleHandler()
        {
        }

        public override Databases.DBObjectCursorBase CreateCursor()
        {
            var cursor = new Finanziaria.CGruppoProdottiCursor();
            return cursor;
        }
    }
}